gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/JdbcConnectionLinker.java

```diff
@@ -24,30 +24,37 @@
 
     @Override
     public void link(JdbcConnectionEntity entity, Bundle bundle, Bundle targetBundle) {
-        String storedPasswordReference = extractVariableName(entity.getPasswordRef());
-        if (isEmpty(storedPasswordReference)) {
-            throw new LinkerException("Password variable specified " + entity.getPasswordRef() + " for JDBC Connection " + entity.getName() + " does not match the expected format (${secpass.<stored_password_name>.plaintext})");
-        }
-
-        Matcher matcher = secpassPattern.matcher(storedPasswordReference);
-        if (matcher.matches()) {
-            // the middle string between secpass and plaintext will be the stored password name
-            String storedPasswordName = matcher.group(1);
-            final StoredPasswordEntity storedPassword =
-                    bundle.getEntities(StoredPasswordEntity.class).values()
-                    .stream()
-                    .filter(e -> e.getName().equals(storedPasswordName))
-                    .findFirst()
-                    .orElse(null);
-            if (storedPassword == null) {
-                throw new LinkerException("Could not find Stored Password for JDBC Connection: " + entity.getName() + ". Password Name: " + storedPasswordName);
+        if (entity.getPassword() != null) {
+            if (entity.getPassword().startsWith("$L7C2$")) {
```

Comment:
Should we add a comment here explaining why of this? or it's a security problem?